plugins/outputs/yandex_cloud_monitoring/README.md

```diff
@@ -8,16 +8,13 @@ https://cloud.yandex.com/services/monitoring
 ```toml
 [[outputs.yandex_cloud_monitoring]]
   ## Timeout for HTTP writes.
-  # timeout = "20s"
+  # timeout = "5s"
```

Comment:
I'm not sure the lower timeout is beneficial here. With some lag you could see a lot of metric retransmission.

Reply:
5s seems pretty huge timeout for monitoring http write handler, isn't it?
It was 5s before. I just changed this config comments to make it have default values as you asked.

Reply:
I believe this is used as the write deadline internally, so it doesn't just refer to 5s idle waiting for the response, but the entire process of writing the metrics in the request. If metrics ever take more than 5s to write, this will retransmit them in a loop until they're dropped, even if the receiver read them.

Reply:
Alright. I will increase the timeout

Reply:
Done